src/articles/components/ArticleView/ArticleImage.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC } from 'react';
+import { DOMAIN } from '../../../common/constants/endpoints';
+import IImage from '../../../common/models/Image';
+import style from './articleView.less';
+
+export interface IProps {
+  image: IImage;
+}
+
+export const ArticleImage: FC<IProps> = ({ image }) => (
+  <section className={style.articleimage}>
+    <img alt={image.description} src={DOMAIN + image.original} />
```

Comment:
I mean if there is one place we should use the original to bask in all it's glory it would be here, and the CSS is already responsive to support basically any size

Reply:
Yeah, I agree that we should utilize the images to their full potential, but then we end up with monstrosities like [this boi here at 8.4 MB](https://online.ntnu.no/media/images/responsive/344acda7-05f2-4d30-94ba-4ffa5a019177.png). The `lg` version is at 950 KB.

Serving 8.4 MB at initial load is not optimal at all, and will break mobile users. Links to the article should not load that large image either.

We can use the `original` image, but it should be loaded after the initial load, or at the very least only on screen sizes which have use for them via `srcSet`, as described in the wiki.